draft-irtf-cfrg-frost.md

```diff
@@ -1128,6 +1127,29 @@ If `vss_verify` fails, the participant MUST abort the protocol, and failure shou
       return 0
 ~~~
 
+We now define how the coordinator and signing participants can derive group info, which is an input into the FROST signing protocol.
+
+~~~
+    derive_group_info(n, t, vss_commitment):
```

Comment:
This is the signing group? As opposed to the algebraic structure?

Reply:
We use group info earlier in the document as an input to each signer with these keys.